### DISCUSSION TITLE
Unsuccessful with Opticore

### DISCUSSION THREAD

**Discussion by kiran-1989** [CodeFHD/OptiCore#12] [General]
Hi,

Many thanks for sharing the code. I am trying to run it however, its unsuccessful. I am not running it correctly. Could you please let me know how to import the python functions?

I have tried running init file from unzipped folder and it has thrown this error:

"ImportError: attempted relative import with no known parent package"

Many thanks for your help.

**Comment by CodeFHD** [comment]
Hi,

this is an addon for the 3D-Software Blender. It sounds like you are trying to run OptiCore standalone from a terminal with python?
It is not designed for that purpose and will not work.

As a guide for that, you can pick any "How to install Blender Addons" tutorial you find online. If you continue to have trouble with that, please give a link to the tutorial you followed and which step is the problem for you.

**Reply by kiran-1989** [reply]
Yes, it works now!! Thank you. That's brilliantly simple. I have another question regarding opticoredemo file. I couldn't get the volumetric scattering as I observe in yours. 

Could you please let me know if I need to change any parameter for the light beam? After downloading the file, I changed to cycles renderer and it didn't show the volumetric scattering.

 Many thanks for your help.

**Reply by CodeFHD** [reply]
The demo file is set up only for LuxCoreRender. It will not work in Cycles.
